website/screens/components/status-light/specs/StatusLightSpecsPage.tsx

```diff
@@ -0,0 +1,47 @@
+import { DxcBulletedList, DxcFlex } from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Figure from "@/common/Figure";
+import DocFooter from "@/common/DocFooter";
+import specsImage from "./images/status-light_specs.jpg";
+import anatomyImage from "./images/status-light_anatomy.jpg";
+
+const sections = [
+  {
+    title: "Specifications",
+    content: (
+      <Figure caption="Status Light design specifications">
+        <Image src={specsImage} alt="Status Light design specifications" />
+      </Figure>
+    ),
+  },
+  {
+    title: "Anatomy",
+    content: (
+      <>
+        <Image src={anatomyImage} alt="Status Light anatomy" />
+        <DxcBulletedList type="number">
+          <DxcBulletedList.Item>Status Light</DxcBulletedList.Item>
+          <DxcBulletedList.Item>Label</DxcBulletedList.Item>
+        </DxcBulletedList>
+      </>
+    ),
+  },
```

Comment:
Should we add a Design Tokens section specifying that it doesn't have tokens? It's a section present on every specifications page.

Reply:
Did Lara said anything about a possible accessibility section?

Reply:
I will ask for her opinion on both topics

Reply:
She thinks that we should add the design tokens section but not the accessibility one (at least for now).